Java/mail-it/src/mail-it/src/main/java/ca/bc/gov/open/rsbc/mailit/mail/MailConfig.java

```diff
@@ -0,0 +1,50 @@
+package ca.bc.gov.open.rsbc.mailit.mail;
+
+import ca.bc.gov.open.rsbc.mailit.mail.mappers.SimpleMessageMapper;
+import ca.bc.gov.open.rsbc.mailit.mail.mappers.SimpleMessageMapperImpl;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+@Configuration
+@EnableWebSecurity
+public class MailConfig {
+
+    @Value("${mail-auth.clientSecret}")
```

Comment:
Could you pls rename these to mail-auth.password and mail-auth.user please? More appropriate to basic auth which uses a user and password compared to oauth which uses id's and secrets, thanks.